service/history/workflow/metrics.go

```diff
@@ -46,68 +46,41 @@ func emitWorkflowHistoryStats(
 	countScope.RecordDistribution(metrics.HistoryCount, historyCount)
 }
 
-func emitWorkflowExecutionStats(
+func emitMutableStateStatus(
 	metricsClient metrics.Client,
+	scope int,
 	namespace string,
```

Comment:
would it make sense to take one metrics.Scope as input instead of these 3?

Reply:
though about that, can be done, but the transaction_impl.go will have more logic
let me take a try